geozero/src/csv/csv_writer.rs

```diff
@@ -477,8 +477,29 @@ POINT(1 45),904 7th Av,05/22/2019 12:55:00 PM,F190051945,Car Fire
             .to_csv()
             .unwrap();
 
-        println!("expected: {}", expected_output);
-        println!("actual: {}", actual_output);
         assert_eq!(expected_output, actual_output);
     }
+
+    #[test]
+    fn waterways() {
+        let input_geojson = r#"{
+            "type": "FeatureCollection",
+            "features": [
+                { "type": "Feature", "properties": { "NAME": "Foo" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 1, 1], [2, 2], [1, 1] ] ] } },
+                { "type": "Feature", "properties": { "NAME": "Bar" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 3, 1], [3, 2], [3, 1] ] ] } }
+            ]
+        }"#;
+
+        let expected_output = r#"geometry,NAME
+"POLYGON((1 1,2 2,1 1))",Foo
+"POLYGON((3 1,3 2,3 1))",Bar
```

Comment:
Without the fix, there's a spurious comma at the beggining of this second row: `,POLYGON((3 1...`